form/form.js

```diff
@@ -1 +1,34 @@
-console.log("Hey");
+function getQuerystring(search) {
+  var pairs = location.search.slice(1).split('&');
+  var result = {};
+
+  pairs.forEach(function(pair) {
+      pair = pair.split('=');
+      result[pair[0]] = decodeURIComponent(pair[1] || '');
+  });
+
+  return result;
+}
+
+function handleFormSubmit () {
+  const interest = document.getElementById("selecInte").value;
```

Comment:
Os campos ainda podem ser alterados de acordo com o que for definidor nos models